neo/Prompt/Commands/Send.py

```diff
@@ -31,15 +31,19 @@ def execute(self, arguments):
         return framework
 
     def command_desc(self):
-        p1 = ParameterDesc('assetId or name', 'the asset you wish to send')
-        p2 = ParameterDesc('address', 'the NEO address you will send to')
-        p3 = ParameterDesc('amount', 'the amount of the asset you wish to send')
-        p4 = ParameterDesc('--from-addr={addr}', 'specify the NEO address you wish to send from', optional=True)
-        p5 = ParameterDesc('--fee={priority_fee}', 'attach a fee to give your tx priority (> 0.001)', optional=True)
-        p6 = ParameterDesc('--owners=[{addr}, ...]', 'specify tx owners', optional=True)
-        p7 = ParameterDesc('--tx-attr=[{"usage": <value>,"data":"<remark>"}, ...]', 'specify unique tx attributes', optional=True)
+        p1 = ParameterDesc('assetId or name', 'the asset (NEO/GAS) to send')
+        p2 = ParameterDesc('address', 'the destination address')
+        p3 = ParameterDesc('amount', 'the amount of the asset to send')
+        p4 = ParameterDesc('--from-addr', 'source address to take funds from (if not specified, take first address in wallet)', optional=True)
+        p5 = ParameterDesc('--fee', 'a fee to give your transaction priority (> 0.001) i.e. --fee=0.01', optional=True)
```

Comment:
It should be `e.g.` rather than `i.e.`

Reply:
Agreed. 
e.g. == for example
i.e. == in other words (more like this is how it should look, not an example)